UPGRADE-4.4.md

```diff
@@ -207,12 +207,44 @@ Translation
 -----------
 
  * Deprecated support for using `null` as the locale in `Translator`.
+ * Deprecated calling the `lint:xliff` command providing pipe file content to STDIN without passing the dash symbol (-) explicitly
```

Comment:
I'd suggest to reorder the sentence (and the similar ones below) this way:
Deprecated accepting STDIN implicitly when using the `lint:xliff` command; use `lint:xliff -` (append a dash) instead to make it explicit.

Then, I don't know if the examples below are very useful.